app/controllers/claims/support/schools/mentors_controller.rb

```diff
@@ -3,11 +3,26 @@ class Claims::Support::Schools::MentorsController < Claims::Support::Application
 
   before_action :set_mentor, only: %i[show remove destroy]
   before_action :authorize_mentor
+  before_action :mentor_form, only: %i[new]
```

Comment:
Can we use the `helper_method` pattern here instead? https://github.com/DFE-Digital/itt-mentor-services/blob/main/app/controllers/claims/schools/claims_controller.rb

`mentor_form` acts more like a memoized instance variable than code that performs an action.

Calling it here is just to make the `@mentor_form` variable available in the view.

Instead, we can make `mentor_form` a helper method and then call that helper method directly in the views. No need to pre-set it here. If we did want to predefine it in the controller, it would make more set to name it `set_mentor_form`.